src/translations/en.json

```diff
@@ -1274,9 +1274,10 @@
             "introduction": "The script editor allows you to create and edit scripts. Please follow the link below to read the instructions to make sure that you have configured Home Assistant correctly.",
             "learn_more": "Learn more about scripts",
             "no_scripts": "We couldn’t find any editable scripts",
-            "add_script": "Add script",
```

Comment:
Please don't rename the key, but just change the translation. Renaming the key means all translations are lost

Reply:
I changed it back. Is it displayed in other languages if only the translation has changed?